### ISSUE TITLE
Failed Exit Code: 126

### ISSUE THREAD

**Issue by dzavodnikov** [cracrayol/vscode-java-pmd#2]
**Describe the bug**

1. Make any changes into the Java file.
2. Save it.
3. Got exception:
```
###################################
Analyzing /Users/dzavodnikov/Git/test-project/src/main/java/pro/zavodnikov/TestFile.java
env: {"CLASSPATH":"/Users/dzavodnikov/Git/test-project/*"}
PMD Command: "/Users/dzavodnikov/.vscode/extensions/cracrayol.java-pmd-0.7.0/bin/pmd/bin/pmd" check --no-progress -f json --no-cache -d "/Users/dzavodnikov/Git/test-project/src/main/java/pro/zavodnikov/TestFile.java" -R "/Users/dzavodnikov/.vscode/extensions/cracrayol.java-pmd-0.7.0/rulesets/quickstart.xml"
stderr:/bin/sh: /Users/dzavodnikov/.vscode/extensions/cracrayol.java-pmd-0.7.0/bin/pmd/bin/pmd: Permission denied
Failed Exit Code: 126
```

**Java PMD Settings**
- pmdPath: `/Users/dzavodnikov/.vscode/extensions/cracrayol.java-pmd-0.7.0/bin/pmd/bin/pmd`
- rulesetPath: `/Users/dzavodnikov/.vscode/extensions/cracrayol.java-pmd-0.7.0/rulesets/quickstart.xml`

**Versions**
 - OS: MacOS 14.4.1
 - vscode: 1.87.2

**Comment by cracrayol** [comment]
I'll try to fix it in the upcoming hours

**Comment by cracrayol** [comment]
I've published v0.7.1 that should resolve the problem.

**Comment by dzavodnikov** [comment]
Thank you very much!
